shaders/src/shading_reflections.fs

```diff
@@ -0,0 +1,23 @@
+/*
+ * screen-space reflection shading
+ */
+vec4 evaluateMaterial(const MaterialInputs material) {
+
+#if defined(MATERIAL_HAS_REFLECTIONS)
+    PixelParams pixel;
+    //getAnisotropyPixelParams(inputs, pixel);
```

Comment:
Remove?

Reply:
Actually that reminds me that I need to fix that :-)